src/js/main.js

```diff
@@ -216,12 +220,8 @@ var SupportKit = Marionette.Object.extend({
             return $.Deferred().resolve(this.user);
         }
 
-        this.user.set(userInfo, {
-            parse: true
-        });
-
         this._throttledUpdate = this._throttledUpdate || _.throttle(this._updateUser.bind(this), 60000);
-        return this._throttledUpdate();
+        return this._throttledUpdate(userInfo);
```

Comment:
yeah, so what about leaving the set in updateUser, but do a set in the other place too and call _updateUser anyway?